packages/odyssey-react-mui/src/Pagination.tsx

```diff
@@ -73,44 +73,47 @@ const PaginationButtonContainer = styled("div")({
   },
 });
 
-export type DataTablePaginationProps = {
+export type PaginationProps = {
   pagination: {
     pageIndex: number;
     pageSize: number;
   };
   setPagination: Dispatch<
     SetStateAction<{ pageIndex: number; pageSize: number }>
   >;
+  lastRow: number;
   totalRows?: number;
   isDisabled?: boolean;
```

Comment:
We need to document these props with JSDoc comments.

Reply:
Done